app/settings/surveys/targeted-surveys/targeted-survey-edit.html

```diff
@@ -180,9 +188,28 @@ <h2 class="stepper-heading">
                                 <span class="stepper-badge">4</span>
                                 <span translate="survey.targeted_survey.publish"></span>
                             </h2>
+                            <div class="alert" ng-show="isActiveStep(4)">
+                                <p translate="survey.targeted_survey.publish_description" translate-values="{questions: targetedSurvey.stepThree.questions.length, numbers:finalNumbers.goodNumbers.length, sms: sms, cost:cost}"></p>
+                            </div>
+                            <div class="form-field switch checked">
+                                <label for="switch4" translate="survey.require_review"></label>
+                                    <p ng-show="editQuestion.require_review" translate="survey.require_review_desc"></p>
+                                    <div class="toggle-switch">
+                                        <input class="tgl" id="switch4" type="checkbox" ng-model="targetedSurvey.stepFour.require_review">
+                                        <label class="tgl-btn" for="switch4"></label>
+                                    </div>
+                            </div>
+                             <div class="form-field switch checked">
```

Comment:
This should also be hidden until you get to step 4

Reply:
Fixed